core/models/instance.py

```diff
@@ -590,13 +623,13 @@ def transaction(cls, status_name, instance, size,
 
     @classmethod
     def create_history(cls, status_name, instance, size,
-                       start_date=None, end_date=None):
+                       start_date=None, end_date=None, activity=None):
```

Comment:
Argument order is different from that in transaction and all `.create_history` above are using arg-order `status_name, activity, instance, size`..